apollo-ios-codegen/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.7
+// swift-tools-version:5.9
```

Comment:
Can we think of any particular reason we this was still at 5.7?

Reply:
I feel like sometime last year I tried to bump us forward on this and someone had an issue with it and we decided not to move forward but I could be thinking of something different. I'll see if I can find a closed PR or something for it.

Reply:
Looks like we had a conversation about this last year and decided to hold off for a bit because it would require Xcode 15 https://apollograph.slack.com/archives/C054R4FEW7L/p1695928304850209

I did update the `apollo-ios` package to 5.9 in January when we did the update of the InstallCLI plugin though - #132

Reply:
Yeah, I don't think we have any reason to wait any longer. Thanks for looking into it @BobaFetters